src/odemis/driver/test/technolution_test.py

```diff
@@ -446,30 +446,43 @@ def test_clock_VAs(self):
                 1 / clock_freq)
 
     def test_resolution_VA(self):
+        """
+        The setter allows only to enter resolutions which are a whole multiple of 4 times the mppc detector shape
+        """
         min_res = self.EBeamScanner.resolution.range[0][0]
         max_res = self.EBeamScanner.resolution.range[1][0]
 
         # Check if small resolution values are allowed
-        self.EBeamScanner.resolution.value = (min_res + 5, min_res + 5)
-        self.assertEqual(self.EBeamScanner.resolution.value, (min_res + 5, min_res + 5))
+        self.EBeamScanner.resolution.value = (self.MPPC.shape[0] * 4, self.MPPC.shape[0] * 4)
+        self.assertEqual(self.EBeamScanner.resolution.value, (self.MPPC.shape[0] * 4, self.MPPC.shape[0] * 4))
 
         # Check if big resolutions values are allowed
-        self.EBeamScanner.resolution.value = (max_res - 200, max_res - 200)
-        self.assertEqual(self.EBeamScanner.resolution.value, (max_res - 200, max_res - 200))
+        self.EBeamScanner.resolution.value = (max_res, max_res)
+        self.assertEqual(self.EBeamScanner.resolution.value, (max_res, max_res))
 
         # Check if VA refuses to set limits outside allowed range
         with self.assertRaises(IndexError):
             self.EBeamScanner.resolution.value = (max_res + 10, max_res + 10)
 
-        self.assertEqual(self.EBeamScanner.resolution.value, (max_res - 200, max_res - 200))
+        self.assertEqual(self.EBeamScanner.resolution.value, (max_res, max_res))
```

Comment:
add comment `# If value out of range was requested, check that value is set to max value possible.`
or should the previous be still set? Some VAs have a .clip method.